pkg/registryserver/keyserver.go

```diff
@@ -30,14 +30,13 @@ type KeyServer interface {
 	// IsReady return if the connection the key server is ready
 	IsReady() bool
 
-	// AttestQuote attests the given quote is valid. Both quote and
-	// publicKey are base64 encoded. The publickey hash part of the quote
-	// must match with the given publicKey.
+	// AttestQuote attests the given quote is valid. The publickey hash part
+	// of the quote must match with the given publicKey.
 	//
 	// Returns true if given quote is valid.
 	// Returns false if verification failed.
 	// In case of other problems, appropriate error gets returned.
-	AttestQuote(ctx context.Context, signerName string, quote []byte, publicKey []byte) (bool, error)
+	AttestQuote(ctx context.Context, signerName string, quote []byte, publicKey []byte, nonce []byte) (bool, error)
```

Comment:
Should the nonce type be just a byte array? I'm just thinking can we encode it in a more specific type?

Reply:
> in a more specific type?

What do you mean by 'specific type'? A Golange type definition for nonce?

Reply:
Yeah, a golang type. Now the `[]byte` type is of course in line with quote and publickey, but maybe using a "real" type would help us set boundaries for the nonce. For example: is it initialized, is it over the minimum length, is it under the maximum length, ...

To be honest this is not really a bug and we don't need to fix it now. Just something to think about :-)